core/src/test/java/org/elasticsearch/search/query/SimpleQueryStringIT.java

```diff
@@ -252,8 +252,7 @@ public void testSimpleQueryStringFlags() throws ExecutionException, InterruptedE
         searchResponse = client()
                 .prepareSearch()
                 .setQuery(
-                        simpleQueryStringQuery("baz | egg*").defaultOperator(Operator.AND).flags(SimpleQueryStringFlag.WHITESPACE,
-                                SimpleQueryStringFlag.PREFIX)).get();
+                        simpleQueryStringQuery("blah | egg*").flags(SimpleQueryStringFlag.WHITESPACE, SimpleQueryStringFlag.PREFIX)).get();
```

Comment:
I'll change this to be a bit more transparent of a test. It's not a very good check right now.